src/core_plugins/console/public/webpackShims/ui-bootstrap-custom.js

```diff
@@ -5,6 +5,33 @@
  * Version: 0.14.2 - 2015-10-17
  * License: MIT
  */
+
+/* @notice
+ * This product bundles angular-ui-bootstrap@0.12.1 which is available under a
+ * "MIT" license.
+ *
+ * The MIT License
+ *
+ * Copyright (c) 2012-2014 the AngularUI Team, https://github.com/organizations/angular-ui/teams/291112
```

Comment:
Should we be updating these year ranges?

Reply:
These copyright notices should reflect the copyright for the version of that tool we're vendoring, so I suspect these particular years should remain as-is.